poppy/instrument.py

```diff
@@ -259,7 +259,7 @@ def calc_psf(self, outfile=None, source=None, nlambda=None, monochromatic=None,
 
         # ---- now at last, actually do the PSF calc:
         #  instantiate an optical system using the current parameters
-        self.optsys = self._get_optical_system(fov_arcsec=fov_arcsec, fov_pixels=fov_pixels,
+        self.optsys = self.get_optical_system(fov_arcsec=fov_arcsec, fov_pixels=fov_pixels,
```

Comment:
Note, this isn't directly related to this PR's main idea, but is an overdue update to some syntax that might as well get included here.